Data Structure & Algorithm/reverseArray.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/*
+ * Complete the 'reverseArray' function below.
+ *
+ * The function is expected to return an INTEGER_ARRAY.
+ * The function accepts INTEGER_ARRAY a as parameter.
+ */
+
+function reverseArray($a) {
```

Comment:
PHP has for that case a good function: `array_reverse` :)
But in this task we have to look into logic

Reply:
👍